hack/test/extensions/extension-patch-filter.jq

```diff
@@ -4,7 +4,7 @@
         "path": "/machine/install/extensions",
         "value": [
             {
-                "image": map(select(. | contains("nvidia-container-toolkit") or contains("nvidia-fabricmanager") | not)) | .[]
+                "image": map(select(. | contains("nvidia") or contains("tailscale") | not)) | .[]
```

Comment:
tailscale keeps on restarting without auth and nvida will be tested with another e2e